.github/workflows/ci.yaml

```
@@ -20,9 +20,9 @@ jobs:
       matrix:
         # Use macos-13 so we'll be on intel hardware and can pull a pre-built wheel
         # When OTIO has an Apple Silicon build we can switch back to macos-latest for that version
-        os: [ubuntu-latest, macos-13, windows-latest]
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
-        otio-version: ["0.17.0", "main"]
+        os: [ubuntu-latest, windows-latest, macos-14, macos-latest]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        otio-version: ["0.17.0", "0.18.1", "main"]
```

```suggestion
        otio-version: ["0.17.0,<0.18.0", "0.18.1,<0.19.0", "main"]
```

I think we also need to change line 43 to use `""`